mobile-browser-based-version/src/components/building_frames/ImageTestingFrame.vue

```diff
@@ -274,48 +274,71 @@ export default {
       IMAGE_WIDTH: null,
   
       model_name: "",
-      FILES: [],
+      FILES: {},
```

Comment:
Once this branch is merged, I'll implement the new file upload manager with the new UploadingFrame